model_mommy/mommy.py

```diff
@@ -238,14 +238,21 @@ def init_type_mapping(self):
             generator = import_if_str(v)
             self.type_mapping[field_class] = generator
 
-    def make(self, _save_kwargs=None, _refresh_after_create=False, **attrs):
+    def make(
+        self,
+        _save_kwargs=None,
+        _refresh_after_create=False,
+        _from_manager=None,
+        **attrs,
```

Comment:
Wondering if this `,` would be the failing tests cause. 🤔

Reply:
Thanks for your comment! As far as I know python has no problem with a trailing comma in a list, no matter the python version. Did I miss something that this can be a problem?

Reply:
Hi @cb109! Did you have a chance to run it locally without this trailing comma? Because I did it and the tests are passing after I removed it from this method and from `_make()` signature.

Reply:
@anapaulagomes You are of course correct and this solved it, I just confirmed with 3.5.6. Thanks and sorry for being a bit stubborn :bowing_man: It seems there was a time where a trailing comma between an unpacked kwargs list was not supported (https://bugs.python.org/issue9232)

Reply:
We should also definitely think about a refactoring for this `make` method. It's almost a god-like implementation hehehe